domains/Nodirbek.json

```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "https://github.com/Piratebay298/"
```

Comment:
![image](https://user-images.githubusercontent.com/87287585/193746857-6ce03378-389c-4b79-b00f-1f630ea30ffc.png)
I just get a white screen.

Reply:
Well fixed it

Reply:
> ![image](https://user-images.githubusercontent.com/87287585/193746857-6ce03378-389c-4b79-b00f-1f630ea30ffc.png) I just get a white screen.

there was missing index.html